index.html

```diff
@@ -0,0 +1,781 @@
+<!DOCTYPE html>
+<html lang="nl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hillegommerdijk 335</title>
+
+    <link rel="stylesheet" href="./styles/style.css">
+
+    <script src="./scripts/main.js" defer></script> 
+</head>
+<body>
+
+    <header>
+            <nav>
+                <div>
+                    <a href="#" class="logo"><img src="./assets/fundawonen-logo.svg" alt=""></a>
+                    <ul class="menu">
+                        <li>
+                            <a href="#">Kopen</a>
```

Comment:
Voor presentatie doeleinden zou ik deze links wel laten linken naar de bewuste live pagina's, zodat het voelt als een echte app. Fake it till you make it!